pkg/wipefs/wipefs.go

```diff
@@ -0,0 +1,72 @@
+package wipefs
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/openshift/lvm-operator/pkg/internal/exec"
+)
+
+var (
+	DefaultWipefs        = "/usr/sbin/wipefs"
+	DefaultDMSetup       = "/usr/sbin/dmsetup"
+	ErrReferenceNotFound = fmt.Errorf("device-mapper reference not found")
+)
+
+type Wipefs interface {
+	Wipe(deviceName string) error
+	RemoveMapperReference(deviceName string) error
+}
+
+type HostWipefs struct {
+	exec.Executor
+	wipefs  string
+	dmsetup string
+}
+
+func NewDefaultHostWipefs() *HostWipefs {
+	return NewHostWipefs(&exec.CommandExecutor{}, DefaultWipefs, DefaultDMSetup)
+}
+
+func NewHostWipefs(executor exec.Executor, wipefs, dmsetup string) *HostWipefs {
+	return &HostWipefs{
+		Executor: executor,
+		wipefs:   wipefs,
+		dmsetup:  dmsetup,
+	}
+}
+
+// Wipe wipes the device only if force delete flag is set
+func (wipefs *HostWipefs) Wipe(deviceName string) error {
+	if len(deviceName) == 0 {
+		return fmt.Errorf("failed to wipe the device. Device name is empty")
+	}
+
+	args := []string{"--all", "--force"}
```

Comment:
Should we try to use non force first, and use force if the policy (see my above comment) allows it? Or is it always mandatory to run force?

Reply:
This was the ask from the beginning, force delete the signatures. Also, see the comment above.